sql-plugin/src/main/scala/com/nvidia/spark/rapids/aggregate.scala

```diff
@@ -121,13 +121,15 @@ object AggregateUtils {
   }
 
   /**
-   * Compute the aggregation modes and aggregate expressions for all aggregation expressions
+   * Bind cuDF aggregate expressions depending on the aggregate expression mode
    * @param aggExpressions the aggregate expressions
    * @param aggBufferAttributes attributes to be bound to the aggregate expressions
```

Comment:
nit: Could we clarify this a bit more? It is hard to tell when aggBufferAttributes is used and when mergeBufferAttributes is used. This is very minor because the code is small enough I can see when it is used easily.

Reply:
done.